container/kubernetes/onboarding/README.md

```diff
@@ -88,16 +88,23 @@ CreateDeployments:
        	GitTag: <MACHINE_CONFIG_TAG_OR_COMMITID_OR_BRANCH>
 ```
 6. Initiate the Deployment process based on three actions; create, update or delete.
+   AND gatewaydeployment yaml file path.
 ```
-python3 deviceonboarding.py --action create 
+python3 deviceonboarding.py --action create --deployfilepath gatewaydeployment.yaml
+OR
+python3 deviceonboarding.py -a create -d gatewaydeployment.yaml
+===================================================================================
+python3 deviceonboarding.py --action update --deployfilepath gatewaydeployment.yaml
 OR 
-python3 deviceonboarding.py --action update
-OR 
-python3 deviceonboarding.py --action delete
+python3 deviceonboarding.py -a update -d gatewaydeployment.yaml
+===================================================================================
+python3 deviceonboarding.py --action delete --deployfilepath gatewaydeployment.yaml
+OR
+python3 deviceonboarding.py -a delete -d gatewaydeployment.yaml
```

Comment:
I think this is confusing for documentation purposes. I would recommend we pick the more human-readable syntax and keep that consistent throughout rather than showing multiple ways of running the same command unless there is good reason to include both for each action.

Reply:
okay